models/dashboards.go

```diff
@@ -111,6 +111,10 @@ func buildIstioAggregations(local, remote string) []Aggregation {
 			Label:       fmt.Sprintf("%s_canonical_revision", local),
 			DisplayName: "Local version",
 		},
+		{
+			Label:       fmt.Sprintf("%s_workload_namespace", remote),
+			DisplayName: "Remote namespace",
```

Comment:
I'm unfamiliar with Istio parlance but to me "Remote" sounds like a namespace in a different cluster. The test above uses `source` as well in `source_workload_namespace`.

Reply:
"Remote" here is really like a synonym for "Peer", nothing related to clusters. We could write "peer" but just as a matter of consistency I won't do it at least in this PR, because this is the terminology we currently have (renaming would impact other places). Note that the "source_workload_namespace" / "destination_workload_namespace" are the prometheus labels to be grouped by (using "source" or "destination" depends whether we are displaying inbound or outbound metrics, and in both case it will be referred to as "Remote namespace" in Kiali)